_config.yml

```diff
@@ -25,7 +25,27 @@ color_image: /assets/img/lineart.png                    # A bit transparent for
 katex: true                                             # Enable if using math markup
 mermaid: true                                           # Enable mermaid-js for sequence and diagrams
 google_analytics:                                       # Tracking ID, e.g. "UA-000000-01"
-cookie_consent: false                                   # To respect the usage of cookies
+cookie_consent: false               # To respect the usage of cookies
+
+# Set which comment system to use
+comments:
+  # 'disqus' or 'cusdis' or also 'utterances' are available
+  provider:            #utterances # or disqus or cusdis
+  
+  
+# You must install utterances github app before use.(https://github.com/apps/utterances)
+# Make sure all variables are set properly. Check below link for detail.
+# https://utteranc.es/
+# for your information, 
+# utterances is open source and free.
+# no ads at all where disqus provides ads and no permission needed when you are using cusdis
+utterances:
+  repo:                "[your repo name here]"
+  issue-term:          "pathname"
```

Comment:
What `issue-term` and `label` do?

Reply:
Issue term is, first, Utterances store the comments as GitHub issues. So, the issue term can be configured in Utterances' official [site](https://utteranc.es). The term can be the title of the blog, or more. Labels, are like the issue labels too.